src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -3,10 +3,12 @@
 
   <UsingTask TaskName="Microsoft.WebAssembly.Build.Tasks.ManagedToNativeGenerator" AssemblyFile="$(WasmAppBuilderTasksAssemblyPath)" />
   <UsingTask TaskName="Microsoft.WebAssembly.Build.Tasks.EmccCompile" AssemblyFile="$(WasmAppBuilderTasksAssemblyPath)" />
+  <UsingTask TaskName="MarshalingPInvokeScanner" AssemblyFile="$(MarshalingPInvokeScannerPath)" />
 
   <PropertyGroup>
     <_WasmBuildNativeCoreDependsOn>
       _WasmAotCompileApp;
+      _ScanAssembliesDecideLightweightMarshaler;
```

Comment:
It might be better to do before compiling AOT? The AOT compilation takes usually quite long time.

Reply:
I agree.